### PR TITLE
Add periodic job ci-node-e2e-containerd-1-7-dra

### PR DESCRIPTION
**Pull request by bart0sh**
Added new periodic job to run DRA tests on Containerd 1.7

### CODE REVIEW

File: config/jobs/kubernetes/sig-node/dynamic-resource-allocation.yaml [hunk 1]
```diff
@@ -78,3 +78,32 @@ periodics:
...
+  - name: ci-node-e2e-containerd-1-7-dra
+    interval: 6h
+    annotations:
+      testgrid-dashboards: sig-node-dynamic-resource-allocation
+      testgrid-tab-name: ci-node-e2e-containerd-1-7-dra
+      testgrid-alert-email: eduard.bartosh@intel.com
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
```
Comment (pohly) [inline config/jobs/kubernetes/sig-node/dynamic-resource-allocation.yaml:92]: `extra_refs` are missing. Without them, the Kubernetes source code will not be available when the job runs.

Comment (bart0sh) [reply config/jobs/kubernetes/sig-node/dynamic-resource-allocation.yaml:92]: This job is a copy of the `ci-node-e2e-crio-dra`, which seems to work correctly. Why this one shouldn't?

**Comment by bart0sh** [comment]
@dims can you approve?

**Pull request merged**